Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM amazoncorretto:11-alpine-jdk
+
+RUN mkdir /data
```

Comment:
It worth using here multistage build. Otherwise your application container would include all the source code (which is unnecessary on production)

Reply:
We are using docker compose for database and server building. Remark about all source code fixed in #6 PR

Reply:
As far as I see, you have not fixed the problem with source code. I meant actually to use multi-stage builds, otherwise the whole code still will be bundled into a single container, while you need only built application.

Reply:
We give up :)